plugin/bertQKVToContextPlugin/qkvToContextPlugin.cu

```diff
@@ -590,8 +594,12 @@ void QKVToContextPluginDynamic::configurePlugin(
 
     const int S = in->max.d[SDIM];
     const int B = in->max.d[BDIM];
-    std::tie(mAlgoBatchedEx1, mAlgoBatchedEx2) = tuneBatchedGemm(B, S, mNumHeads, mHeadSize);
-    gLogVerbose << "QKV Plugin - Selected Algos for batch gemms: " << mAlgoBatchedEx1 << ", " << mAlgoBatchedEx2 << "\n";
+    if ((S != mS) || (B != mB)) {
```

Comment:
@ryanleary also need to check if `in->max.d[SDIM/BDIM]` was changed, which also takes the if case.

Reply:
Can you clarify what extra check is needed here? I'm not following. `SDIM = 0` so `in->max.d[SDIM] == in->max.d[SDIM/BDIM]`.